packages/backend/src/tenants/service.ts

```diff
@@ -88,8 +91,17 @@ async function createTenant(
       idpConsentUrl
     })
 
+    const settings = await deps.tenantSettingService.create(
+      {
+        tenantId: tenant.id,
+        setting: TenantSetting.default()
+      },
+      { trx }
+    )
+
     await trx.commit()
 
+    tenant.settings = settings
```

Comment:
Is this to set it on the cache? If so, we would also need to change the update method. 

Or we just have a separate cache for tenant settings (or no cache at all initially)

Reply:
Ah this is some leftover from experimenting. Nice catch. Will remove this. cache can probably be separate for tenant and for settings.